src/features/cookies/helpers/setMarketingParams.native.test.ts

```diff
@@ -73,20 +94,27 @@ describe('setMarketingParams', () => {
 
       jest.runOnlyPendingTimers()
 
-      expect(setUtmParamsSpy).toHaveBeenNthCalledWith(1, UTM_PARAMS)
+      await act(() => {}) // Because of the "await" in the setTimeout
+
+      // We don't test campaign_date because even with mockDate, the setTimeout creates delay.
```

Comment:
c'est dommage on ne test plus la date

as-tu essayé quelque chose du style ?

```suggestion
      // We don't test campaign_date because even with mockDate, the setTimeout creates delay.
      expect(Math.round(Number(await storage.readString('campaign_date')) / 100).toString()).toEqual(Math.round(new Date().getTime() / 100).toString()
```

Reply:
ça marche comme ça, merci